e2e_samples/unstructured_data/scripts/evaluate_experiments.py

```diff
@@ -83,7 +83,7 @@
     if args.metadata_paths is not None:
         metatdata_paths = []
         for p in args.metadata_paths:
-            metatdata_paths.append(RepoPaths.run_outputs.joinpath(p))
```

Comment:
The orchestrator now has its own config that controls the expected directories so we use that as the source of truth